src/server/services/wms/qgswmsparameters.h

```diff
@@ -367,6 +369,40 @@ namespace QgsWms
        */
       int heightAsInt() const;
 
+      /**
+       * Returns SRCWIDTH parameter or an empty string if not defined.
+       * \returns srcWidth parameter
+       * \since QGIS 3.8
```

Comment:
This should have been `3.4.7` in master because you are backporting to 3.4.

Reply:
you're right